jpo-ode-plugins/src/main/java/us/dot/its/jpo/ode/plugin/j2735/builders/BrakeSystemStatusBuilder.java

```diff
@@ -29,8 +31,17 @@ private BrakeSystemStatusBuilder() {
    public static J2735BrakeSystemStatus genericBrakeSystemStatus(JsonNode brakesStatus) {
       J2735BrakeSystemStatus genericBrakesStatus = new J2735BrakeSystemStatus();
 
-      genericBrakesStatus.setWheelBrakes(BitStringBuilder.genericBitString(brakesStatus.get("wheelBrakes"),
+      JsonNode wheelBrakes = brakesStatus.get("wheelBrakes");
+      // Check if wheelBrakes was not set (00000), if it wasn't set it to unavailable
+      if (wheelBrakes.asText().trim().equals("00000")) {
```

Comment:
Is it possible to receive 00000 for wheelbrakes from a vehicle? https://github.com/usdot-fhwa-stol/carma-messenger/blob/f2bb8f14fd4884f619598b7739136fe81dbe2f7b/carma-messenger-core/cpp_message/src/BSM_Message.cpp#L192

Reply:
or any other values other than "10000", "01000", "00100", "00010", "00001"?

Reply:
Turns out it is! We are seeing it in the BSMs we are receiving in our CDOT deployment of the ODE.

Reply:
We get many 00000s

Reply:
good to know, thanks!